googleapiclient/http.py

```diff
@@ -1120,10 +1120,6 @@ def _serialize_request(self, request):
     g.flatten(msg, unixfrom=False)
     body = fp.getvalue()
 
-    # Strip off the \n\n that the MIME lib tacks onto the end of the payload.
-    if request.body is None:
```

Comment:
I'm not sure.  But, are you reading these line deletions as additions?  This change is to remove the stripping, not to add it.